lib/sparrow/pools_warden.ex

```diff
@@ -70,7 +70,7 @@ defmodule Sparrow.PoolsWarden do
     )
   end
 
-  @spec init(any) :: {:ok, :ok}
+  @impl GenServer
```

Comment:
based on: https://hexdocs.pm/elixir/GenServer.html#module-example
how about: `@impl true`?

Reply:
That's what I wrote at first as well, but Rafal suggested changing it this way to be more explicit and clear (I believe)

Reply:
I shall not argue with that ;)

Reply:
@aleklisi @kmakiela Just FYI, according to official [doc](https://hexdocs.pm/elixir/Module.html#module-impl), both `@impl true` and `@impl GenServer` are just fine in this case. The only difference is that `@impl true` checks any used behaviors, while `@impl SomeBehavior` checks only `SomeBehavior` specifically, which simply makes it more explicit. In a module that implements only one behavior, those are effectively equivalents, but if someone would add another behavior, I doubt that someone would remember to change this, and thus override warnings could not work as expected. I know this is highly unlikely, but I don't see any reason why not to use `@impl GenServer` which is just much safer in this regard.